client/src/tests/setup.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
This file will be run before all tests automatically and is intended to be a convenient way to reuse query mocking for all tests. I'm a little worried that it'll happen a little too magically and devs won't know where the test data is coming from, but hopefully developers will find this file and see this comment.

Reply:
Updated the readme!